website/src/components/Messages/AdminMessageTable.tsx

```diff
@@ -106,5 +119,17 @@ const DateDiff = ({ children }: { children: string | Date | number }) => {
 };
 
 export const AdminMessageTable = (props: StrictOmit<DataTableProps<Message>, "columns">) => {
+  const { trigger: deleteMessage } = useSWRMutation("/api/admin/user_messages", (path, args: { arg: string }) =>
```

Comment:
Why you change the swr key here?

Reply:
Because that's how swr is supposed to work.  We are modifying the key /api/admin/user_messages  (It can affect the child messages too). 
Without this, the UI does not correctly update.

Reply:
This is now extracted to a custom hook